packages/web-app-files/src/components/AppBar/SidebarToggle.vue

```diff
@@ -5,30 +5,37 @@
     :aria-label="toggleSidebarButtonLabel"
     appearance="raw"
     class="oc-my-s oc-p-xs"
-    @click.stop="toggleSidebar"
+    @click.stop="toggleSideBar"
   >
     <oc-icon name="side-bar-right" :fill-type="toggleSidebarButtonIconFillType" />
   </oc-button>
 </template>
 
 <script>
-import { mapState, mapActions } from 'vuex'
+import { bus } from 'web-pkg/src/instance'
+import { SideBarEventTopics } from '../../composables/sidebar'
```

Comment:
We might rename this composable to sideBar ad well

Reply:
You mean just renaming the folder to `sideBar`?

Reply:
Yes

Reply:
Ok, understood. Will do :-)

Reply:
done